www/_posts/2022-05-30-cordova-electron-release-3.1.0.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Electron 3.1.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Electron 3.1.0`! This is one of Cordova's supported platforms for building Electron applications.
+
+* [cordova-electron@3.1.0](https://www.npmjs.com/package/cordova-electron)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove electron
+cordova platform add electron@3.1.0
+```
+
+**To install:**
+
+```bash
+cordova platform add electron@3.1.0
+```
```

Comment:
Doesn't this bit usually go above the "Release Highlights" heading?

Reply:
Yeah, seems I some how mixed it up.. I moved it up.